test/ProjectService.Validation.UnitTests/CreateRoleRequestValidatorTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using FluentValidation;
+using FluentValidation.TestHelper;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Validation.UnitTests
+{
+    public class CreateRoleRequestValidatorTests
+    {
+        private IValidator<CreateRoleRequest> validator;
+        private CreateRoleRequest roleRequest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            validator = new RoleValidator();
+
+            roleRequest = new CreateRoleRequest
+            {
+                Name = "Lead Tester Grand Manager 2nd",
+                Description = "New role in DigitalOffice"
+            };
+        }
+
+        [Test]
+        public void ShouldHaveValidationErrorWhenRoleNameIsEmpty()
+        {
+            validator.ShouldHaveValidationErrorFor(x => x.Name, "");
+        }
+
+        [Test]
+        public void ShouldHaveValidationErrorForWhenRoleNameIsTooLong()
+        {
+            roleRequest.Name += roleRequest.Name.PadLeft(81);
+
+            validator.TestValidate(roleRequest).ShouldHaveValidationErrorFor(request => request.Name)
+                .WithErrorMessage("Role name is too long.");
```

Comment:
I think it's best not to use WithErrorMessage, because if the message changes, you'll have to change everything in two places. Better to clarify this point with Spartak